### PR TITLE
Unit 4 - Higher order programming - Eddie Tong

### PR DESCRIPTION
**Pull request by Edison-Tong**
Screenshot 2024-07-29 at 5 34 47 PM

### CODE REVIEW

File: Unit 4/higher-order-programming/higher-order-programming-exercise/script.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+calculator(3, 5, multiplication);
+calculator(7, 5, addition);
+calculator(35, 8, subtraction);
+calculator(681, 3, division);
+
+// Poor man's forEach Exercise
+
+function forEach(array, func) {
+  for (let i = 0; i < array.length; i++) {
+    console.log(func(array[i]));
```
Comment (AcquirEdge) [inline Unit 4/higher-order-programming/higher-order-programming-exercise/script.js:35]: Not exactly... "forEach()"calls "func" on each element of the array. It doesn't log the result to the console.

File: Unit 4/higher-order-programming/higher-order-programming-exercise/script.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  }
+}
+
+let yellIt = function (string) {
+  return string.toUpperCase();
```
Comment (AcquirEdge) [inline Unit 4/higher-order-programming/higher-order-programming-exercise/script.js:40]: yellIt should log "string" in all caps, not return it.

**Review by AcquirEdge — CHANGES_REQUESTED** [review]
Some small changes to make.

Other than my comments, everything looks good!

File: Unit 4/higher-order-programming/higher-order-programming-exercise/script.js (hunk 1 above)
Comment (Edison-Tong) [reply Unit 4/higher-order-programming/higher-order-programming-exercise/script.js]: Okay I think I got it this time

**Review by AcquirEdge — APPROVED** [review]
Look sgood

**Pull request merged**